cucumber/api/features/whoami.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Who Am I
+
+  Scenario: Audit entry
+
+    Given I am a user named "alice"
+    And I can GET "/whoami"
```

Comment:
this looks like a `When` (and also remove the `can`)

Reply:
I'll replace the 'can' to 'successfully'... what do you think ?